cmd/semantic-release/semantic-release.go

```diff
@@ -67,15 +70,27 @@ func main() {
 	case "up":
 		logger.Info(colorYellow + "\nSemantic Version just started the process...\n\n" + colorReset)
 
-		semantic := newSemantic(logger, upgradeVersionCmd, gitHost, groupName, projectName, username, password, upgradePyFile)
-
-		if err := semantic.GenerateNewRelease(); err != nil {
-			logger.Error(err.Error())
-			os.Exit(1)
+		fmt.Println(*gitHost)
```

Comment:
should we change to logger? Maybe we can move it to the previous log Info.

Reply:
@julianogalgaro 

I've removed it.

It was a useless print.